examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+# Deferred Processing and Transactions
+
+## Introduction
+[Deferred processing](https://docs.pachyderm.com/latest/how-tos/deferred_processing/) 
+is a Pachyderm technique for controlling when data gets processed.
+Transactions are Pachyderm feature for assuring data from different repos get processed together.
+
+This is an example which uses a simple DAG, 
+based on our opencv example, 
+to illustrate using deferred processing along with transactions.
+
+
+
+## Pipelines
+
+![Example DAG](./example_dag.png)
+
+The DAG shown is a simple elaboration on the opencv example,
+with pipeline and repo names chosen to avoid collisions with that example 
+if installed in the same cluster.
+
+The functionality is, of course, different.
```

Comment:
```suggestion
However, the functionality is different.
```

Reply:
I was taught to never begin a sentence with "however", as it's only used to introduce another clause.  So I'll just rewrite this to remove the "of course".